HIP-xxxx.md

```diff
@@ -0,0 +1,163 @@
+
+#### Domain Parking
+
+The parking TXT entry indicates that the domain is for sale.
+
+```
+TXT parking=[String]
+```
+
+#### Contact or Profile Information
+
+```
+TXT email=[Email]
+TXT phone=[Tel]
+```
+
+```
+TXT discord=[String]
+TXT facebook=[String]
+TXT github=[String]
+TXT instagram=[String]
+TXT linkedin=[String]
+TXT telegram=[String]
+TXT tiktok=[String]
+TXT twitter=[String]
+TXT weixin=[String]
+TXT whatsapp=[Tel]
+TXT youtube=[String]
+```
+
+#### Avatar
+
+```
+TXT avatar=[ URL | skylink | ipfsLink ]
+```
+
+#### Domain Caption
+
+A caption is especially useful for identities or as a sales pitch. It can also be used to add more context to a domain (e.g. NFT's).
+
+```
+TXT caption=[String]
+```
+
+#### Payment / Wallet Alias
+
+Although [HIP-0002](https://github.com/handshake-org/HIPs/blob/master/HIP-0002.md) is a much better approach, TXT entries could also be used to provide static wallet information.
+
+```
+TXT wallet=[Chain]:[Address]
+```
+
+## Valid TXT Standard Examples
+
+#### Parking Examples
+
+```
+TXT parking
+TXT parking=10000 HNS
+TXT parking=USD 1000
+TXT parking=USD 999.99
+TXT parking=999,99 EUR
+TXT parking=$199 (Old price $329)
+TXT parking=Please send an offer via email
+```
+
+Every string is valid, but usually a price is provided. Data providers should be aware that prices could be devided by a dot or a comma and can have decimal points.
+
+No matter what data is provided, as long as the parking TXT entry is added, the domain will be seen as "For Sale". It's up to the site owner or app to make sense of the data. Remember: It's impossible to validate the data via HSD.
+
+#### Contact Information Examples
+
+```
+TXT email=hello@handshake.org
+TXT phone=+1(310)3015800
+```
+
+The phone number should be prefixed with a plus sign and country code. Hyphens, dots and empty spaces are allowed.
+
+```
+TXT discord=handshake#1234
+TXT facebook=handshake
+TXT github=handshake
+TXT instagram=handshake
+TXT linkedin=handshake
+TXT telegram=handshake
+TXT tiktok=handshake
+TXT twitter=handshake
+TXT weixin=DDvRyffEabcdefGh123e
+TXT whatsapp=907123987234
+TXT youtube=handshake
+```
+
+For most social media profiles it's enough to provide the handle. WeChat (weixin) needs a chat ID. WhatsApp links will only work without a prefix, therefore it's best to omit them. We can always add or remove networks if needed.
+
+#### Avatar Examples
+
+```
+TXT avatar=https://handshake.org/images/landing/logo-dark.svg
+TXT avatar=sia:CABAB_1Dt0FJsxqsu_J4TodNCbCGvtFf1Uys_3EgzOlTcg
+TXT avatar=ipfs:bafybeidd2gyhagleh47qeg77xqndy2qy3yzn4vkxmk775bg2t5lpuy7pcu
+```
+
+#### Domain Caption Example
+
+```
+TXT caption=The Wheel of Fortune represents the end of one cycle and the beginning of the next
+```
+
+#### Payment / Wallet Alias Examples
+
+```
+TXT wallet=hns:hs1qnkr3cwf9ldp7qpq2csuuyxf0znj0lwhmawhlal
+TXT wallet=btc:12dRagNcdxK39288NjcDV4GX7rMsKCGn6B
+TXT wallet=eth:0x28346f1ec065eea239152213373bb58b1c9fc93b
+```
+
```

Comment:
Can't have threads without a like afaik so continuing here.

@rithvikvibhu:
>Idea: a field called meta with value of a https url. The page must follow the same standard parsing of TXT fields, but since it's served by a website, it can be dynamic. Of course on chain records are possible, and preferred in cases, but with pointing to a website (secured by CA/DANE), it's possible to do a lot more things using existing field standards:
>- Not constrained to 512 bytes (blockchain dns size limit), also leaves more room for other records
>- listing price changes based on page views/time
>- dynamic wallet addresses (though there's HIP-2 for this)

@Falci:
>>Idea: a field called meta with value of a https url...

>Kinda of .well-known directory, no?

@rithvikvibhu:
Kinda, but it need not be on the same website. I could point it to `https://parking.sinpapeles/hipXXX/tld` without giving parking.sinpapeles to host any website like how an A record would do.

Reply:
That's actually a neat idea and would provide much more flexibility to all domain owners. Plus would save a lot of transaction fees in the future. I really like that you could point to any URL and even make it dynamic! Could even be an alternative for the `.well-known` directory.

I think that's a great solution and much better than handling dozens of TXT records

Reply:
Although I really really like the idea, there's one concern I have. Which is also the reason I have not included it the updated PR, yet.

This method forces a website to download a file and process it. My concern is that a bad actor could create extremely large files with junk data and the website would download and process it. This can be bad bandwidth wise but also server processing power could be at risk. In the worst case scenario this could be used for a DDoS attack.

Sure, there could be guardrails put in place on the webserver, to stop if a file get's too large. But that doesn't feel lightweight and easy to me.

TXT records on the other hand are limited in their size and easy to process.

Any thoughts on this? I do welcome polite pushback

Reply:
I still very much like the idea to provide data via one link as a TXT record. 
Although, probably better to create its own HIP for it and reference all TXT naming standards. Therefore resolving this here.

Reply:
Oh no I missed this thread in the emails (march and may?!)

Yeah that makes sense, can go in a future proposal. Seeing how there's already 2 that use TXT and one draft for authentication that may need TXT, we'll need something to "scale" sooner or later.